plugins/@grouparoo/iterable/src/lib/export/exportProfile.ts

```diff
@@ -111,6 +116,7 @@ export const sendProfile: ExportProfilePluginMethod = async ({
       );
     }
 
+    payload["mergeNestedObjects"] = true;
```

Comment:
This implies that in all cases, we can send incomplete updates to an Iterable object and we will merge the existing data with what we have.  Cool!

What doest it mean when we want to remove a piece of data?  For example, in your test you could set `addres.city` and then remove it... would it be removed from Itterable?